### PR TITLE
Application permissions - update permissions table

### PR DESCRIPTION
**Pull request by cristobal-buenrostro**
Application permissions updated.

Work item:
https://dev.azure.com/1edu/Homeroom/_workitems/edit/152345

### CODE REVIEW

**Review by isvargasmsft — APPROVED** [review]

**Comment by Lauragra** [comment]
@cristobal-buenrostro , please add a changelog entry for the new permissions before we merge this PR. Thanks.

**Comment by Lauragra** [comment]
@cristobal-buenrostro , please include changelog entries to call out that support for application permissions was added for these APIs.

**Comment by cristobal-buenrostro** [comment]
> @cristobal-buenrostro , please include changelog entries to call out that support for application permissions was added for these APIs.

@Lauragra   , changelog added

File: changelog/Microsoft.EducationAssignment.json [hunk 1]
```diff
@@ -1,5 +1,105 @@
...
   "changelog": [
+    {
+      "ChangeList": [
+        {
+          "Id": "b8528e3b-c132-4de7-a673-2f624927e011",
+          "ApiChange": "Permission",
+          "ChangedApiName": "Update educationOutcome",
+          "ChangeType": "Addition",
+          "Description": "Added support for application permissions to the [Update educationOutcome](https://docs.microsoft.com/en-us/graph/api/educationoutcome-update?view=graph-rest-beta).",
+          "Target": "educationOutcome"
```
Comment (isvargasmsft) [inline changelog/Microsoft.EducationAssignment.json:11]: Changelog: @cristobal-buenrostro please follow examples 14 and 15 here: https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/211742/Changelog?anchor=changelist-record-examples. 

Specifically, please take a look at the ChangedApiName values.

**Review by isvargasmsft — CHANGES_REQUESTED** [review]
Hi @cristobal-buenrostro, just 1 more detail, please follow the examples 14 and 15 in the [changelog guide](https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/211742/Changelog?anchor=changelist-record-examples).

File: changelog/Microsoft.EducationAssignment.json (hunk 1 above)
Comment (cristobal-buenrostro) [reply changelog/Microsoft.EducationAssignment.json]: done

**Review by isvargasmsft — APPROVED** [review]

**Pull request merged**